File: Foundation/Arrays/span.java
import java.util.Scanner;
public class span
{
    public static Scanner scn = new Scanner(System.in);
    public static void span(int[] arr)
    {
        int max = (int)-1e8;
        int min = (int)1e8;
        int span;
        for(int i = 0; i < arr.length; i++)
        {
            if(arr[i] < min)
            {
                min = arr[i];
            }
            if(arr[i] > max)
            {
                max = arr[i];
            }
        }
        span = max - min;
    }

    public static void main(String[] args)
    {
        int n = scn.nextInt();
        int[] arr = new int[n];
        for(int i = 0; i < arr.length; i++)
        {
            arr[i] = scn.nextInt();
        }
        span(arr);
    }
}

File: Foundation/Arrays/span.cpp
#include<iostream>
#include<vector>
using namespace std;

void span(vector<int>& arr)
{
    int max = -1e8;
    int min = 1e8;
    int result;
    for(int i = 0; i < arr.size(); i++)
    {
        if(arr[i] > max)
        {
            max = arr[i];
        }
        if(arr[i] < min)
        {
            min = arr[i];
        }
    }

    result = max - min;
    cout<<result;
}

int main()
{
    int n; 
    cin >> n;
    vector<int> arr(n);
    for(int i = 0; i < arr.size(); i++)
    {
        cin>>arr[i];
    }
    span(arr);
    return 0;
}